### ISSUE TITLE
Great tool thanks

### ISSUE THREAD

**Issue by Shuunen** [Harpseal/TomatoTimerWPF#4]
Starring is the only way of feedback, so I'm opening this issue to say thanks 👍 ❤️

**Comment by SMUsamaShah** [comment]
Same! Thanks.

**Comment by Openheads** [comment]
I am new too GitHub, can i actually use this on my windows desktop?

**Comment by Shuunen** [comment]
@Openheads yes go to releases : https://github.com/Harpseal/TomatoTimerWPF/releases/ 

download the `TomatoTimerWPF_bin_190214_2.7z` file, un-compress & start the program :+1:

**Comment by shenggaowei** [comment]
thanks
